src/core/api/public_api.ts

```diff
@@ -307,6 +308,11 @@ class Player extends EventEmitter<IPublicAPIEvent> {
     reloadPosition?: number;
   };
 
+  /**
+   * Store last value of autoPlay, coming from loadVideo or onReloadMediaSource.
```

Comment:
`onReloadMediaSource` is a concept from the `init` (`src/core/init/media_source_content_initializer.ts`) whereas here we are in what is another part of the code, the API.

For the API, all it sees is a `"reloadingMediaSource"` event coming from the init.

So even if this is just a comment, I'm not too comfortable about referencing that init function here, both because someone reading the API code won't be able to quickly understand what it is a reference to, and mostly because the `init` inner code might completely change without us even thinking about updating this comment.

Reply:
Right I understand, do you prefer referencing the `"reloadingMediaSource"` event or keeping details about where the value comes from out of this comment?